extension/source/Controllers/utils.ts

```diff
@@ -12,14 +18,9 @@ export function timeout(duration: number): Promise<void> {
 export const createRandomId = (): string => Math.random().toString(36).slice(2);
 
 export const getUserLanguage = (): string => {
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  let userLanguage =
-    (window.navigator as any).userLanguage ||
```

Comment:
I could not find any relevant documentation for `window.navigator.userLanguage` so I just removed it.